Makefile

```diff
@@ -15,6 +15,14 @@ PHASEB           = build/phaseB
 PHASEC           = build/phaseC
 RELEASE_DIR      = build/release
 
+FLATNESS_THRESHOLD = 10
+ifeq ($(FLATNESS_THRESHOLD),-1)
+  # If we allow infinite flatness, then get rid of the system stack
+  # limit for node
+  TMPNODE:=$(NODE)
+  NODE=ulimit -s unlimited; $(TMPNODE)
```

Comment:
@blerner do you know what the equivalent of this would be on Windows (not urgent)